modules/ossm-exposing-a-gateway-to-traffic-outside-the-cluster-using-openshift-routes.adoc

```diff
@@ -39,20 +39,29 @@ spec:
     name: <gateway_name>
     weight: 100
   wildcardPolicy: None
-  # ...
 ----
 
-. Verify that httpbin service can be accessed from outside the cluster through the ingress router. Ensure that you set the `INGRESS_HOST` variable appropriately for the environment that your cluster is running in. 
+. Verify that `httpbin` service can be accessed from outside the cluster through the ingress router. Ensure that you set the `INGRESS_HOST` variable appropriately for the environment that your cluster is running in. 
+
+.. If the cluster runs on AWS, run the following command to set the `INGRESS_HOST` variable:
 +
 [source,terminal]
 ----
-# If the cluster is running on AWS:
-INGRESS_HOST=$(oc get service router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
-
-# If the cluster is running on GCP or Azure:
-INGRESS_HOST=$(oc get service router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-
-curl -s -I -H Host:httpbin.example.com http://$INGRESS_HOST/headers
+$ INGRESS_HOST=$(oc get service router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
 ----
 
-. Verify that the terminal returns `HTTP/1.1 200 OK` which indicates that the request was successful.
+.. If the cluster runs on GCP or Azure, run the following command to set the `INGRESS_HOST` variable:
++
+[source,terminal]
+----
+$ INGRESS_HOST=$(oc get service router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+----
+ 
+.. Send a `curl` request to the `httpbin` service using the host of the ingress router:
++
+[source,terminal]
+----
+$ curl -s -I -H Host:httpbin.example.com http://$INGRESS_HOST/headers
+----
++
+Verify that the response has the `HTTP/1.1 200 OK` status, which indicates that the request was successful.
```

Comment:
This is arguably another instruction for users to follow and should be its own step:
```suggestion
. Verify that the response has the `HTTP/1.1 200 OK` status, which indicates that the request was successful.
```

Reply:
done